general/server/plugins/index.md

```diff
@@ -33,61 +33,157 @@ All plugins hosted on the repository can be built from source and manually added
 
 ### Official Plugins
 
-#### anime
+#### Anime
 
 [![](https://img.shields.io/github/languages/top/jellyfin/jellyfin-plugin-anime.svg)](https://github.com/jellyfin/jellyfin-plugin-anime)
 [![](https://img.shields.io/github/contributors/jellyfin/jellyfin-plugin-anime.svg)](https://github.com/jellyfin/jellyfin-plugin-anime)
 [![](https://img.shields.io/github/license/jellyfin/jellyfin-plugin-anime.svg)](https://github.com/jellyfin/jellyfin-plugin-anime)
 
-#### autoorganize
+Manage your Anime in Jellyfin. Supports several different metadata providers and options for organizing your collection.
+
+#### Auto Organize
 
 [![](https://img.shields.io/github/languages/top/jellyfin/jellyfin-plugin-autoorganize.svg)](https://github.com/jellyfin/jellyfin-plugin-autoorganize)
 [![](https://img.shields.io/github/contributors/jellyfin/jellyfin-plugin-autoorganize.svg)](https://github.com/jellyfin/jellyfin-plugin-autoorganize)
 [![](https://img.shields.io/github/license/jellyfin/jellyfin-plugin-autoorganize.svg)](https://github.com/jellyfin/jellyfin-plugin-autoorganize)
 
-#### emailnotifications
+Automatically organize your media
```

Comment:
I'd use full sentences for all the descriptions.

Reply:
I’ll probably need some help with this, as I don’t actually know what some of these plugins do.

Reply:
It's fine, we can always do this in another pull request.